.babelrc

```diff
@@ -1,9 +1,17 @@
 {
-  "presets": ["es2015"],
-  "plugins": [
-    "transform-flow-strip-types",
-    "transform-es3-property-literals",
-    "transform-es3-member-expression-literals",
-    ["transform-es2015-for-of", {"loose": true}]
```

Comment:
Yeah we do, it's in a load of places: 

https://github.com/krakenjs/cross-domain-utils/blob/master/src/index.js#L228